README.md

```diff
@@ -18,6 +18,23 @@ the config file `ledger2beancount.yml` and if that is not found for
 file for the variables.
 
 
+Dependencies
+------------
+
+ledger2beancount uses several Perl modules:
+
+   * Carp::Assert
```

Comment:
drop the leading spaces here, they're not needed for markdown lists, and you're just one space short of the text being considered preformatted ;-)